### ISSUE TITLE
Can`t use GPU on testing

### ISSUE THREAD

**Issue by CowInSiberia** [mit-han-lab/temporal-shift-module#150]
Environment:
Anaconda3
Python: 3.7
PyTorch: 1.3.1
torchvision: 0.4.2
cudatoolkit: 11.0
Server: Linux
GPU: 4*Nvidia 2080Ti

Trouble:
After installing the environment and downloading the pretrained models, try to run 'test_tsm_kinetics_rgb_8f.sh' and get some trouble.
When change 'map_location=None' to 'map_location=lambda storage, loc: storage.cuda(1))',something wrong with '_cuda_getDevice' and test
![singlegpu](https://user-images.githubusercontent.com/31331585/96945086-3c94a780-150f-11eb-99fe-c119e1a42011.png)
![traceback1](https://user-images.githubusercontent.com/31331585/96945087-3dc5d480-150f-11eb-972a-7b89bb998297.png)

When change 'map_location=None' to 'map_location={'cuda:1':'cuda:0'})',something wrong with 'torch.cuda.is_available()'.
![multigpu](https://user-images.githubusercontent.com/31331585/96945273-deb48f80-150f-11eb-8491-5402d572fe9f.png)
![traceback2](https://user-images.githubusercontent.com/31331585/96945275-dfe5bc80-150f-11eb-84d2-8d09c74506fb.png)

If I change map_location to torch.device('cpu'), test will run but cant get a result because I havent upload the test video and labels and annotations to the server. This is not a problem but Im wondering why cant I use gpu to do the test, if I can only run test.sh with cpu then I cant train my own dataset.

Really appreciate any help!

**Comment by yjp942749578** [comment]
try to install CUDA and cuDNN

**Comment by CowInSiberia** [comment]
> try to install CUDA and cuDNN

I find out that I misinstalled the cpu version pytorch and after installing cuda version pytorch, test_model works.
Thank you very much!!!
